src/scripts/main.js

```diff
@@ -2,7 +2,16 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
-
-// write your code here
+people.forEach(person => {
+  document.querySelector('.dashboard').children[0]
```

Comment:
querySelector краще зробити за межами циклу і тег tbody дістати через `.dashboard tbody`, замість   `children[0]`
```suggestion
  document.querySelector('.dashboard tbody')
```